n2_cooling/cooling_without_flowmeter.py

```diff
@@ -0,0 +1,194 @@
+        else:
+            self.socket = None
+
+        # Set up temperature log file
+        self.temp_type = np.dtype([
+            ('timestamp', 'u4'),
+            ('temperature_box', 'f4'),
+            ('temperature_dut', 'f4'),
+            ('humidity_dut', 'f4'),
+        ])
+        self.output_file = tb.open_file('measurement_' + today + current_time + '.h5', 'a')
+        if '/temperature' in self.output_file:  # Table already exists
+            self.temp_table = self.output_file.root.temperature
+        else:
+            self.temp_table = self.output_file.create_table(self.output_file.root,
+                                                            name='temperature',
+                                                            description=self.temp_type,
+                                                            filters=tb.Filters(complevel=5, complib='blosc')
+                                                            )            
+
+    def __del__(self):
+        self.output_file.close()
+
+    def mean(self,list):
+        sum_of_list = 0
+        for i in range(len(list)):
+            sum_of_list += list[i]
+        average = sum_of_list/len(list)
+        return average
+
+
+    def get_temps(self):
+        '''
+        This func will ask for data by writing "R" in the Serial port. It will receive a Array with 3 elements.
+        ([tempNTC,tempSHT, humidityDHT])
+        '''
+        # start process in arduino with command 'R'
+        ser_arduino.write(bytes(b'R'))
+       
+        time.sleep(1)
+
+        # output is from type bytes. Convert to string
+        read_values = ser_arduino.readline().decode("utf-8")
+        values = read_values.split(" ")
+
+
+        tempNTC = values[0]
+        tempSHT = values[1]
+        humidSHT = values[2]
+
+
+        return [tempNTC, tempSHT, humidSHT]
+
+
+    def PID_controller(self, user_input):
+       
+        logging.info('Starting cooling process...')
+
+        with open('measurement_' + today + current_time + '.txt', 'w') as f:
+            '''
+            This function will include thw hole PID controller. It uses mainly the simple_PID package
+            '''
+
+            write_flg = 1
+            rst_flg = 0
+
+            while True:
+
+                # printing the measurement
+                measurement=self.get_temps()
+               
+                print("VALVE: ", 0, "%")
+                print('TEMP NTC: ', measurement[0], "°C")
+                print("TEMP SHT: ", measurement[1], "°C")
+                print("HUMID: ", measurement[2], "% \r\n")#, humids = self.get_temps()
+
+                print('_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n')
+
+                # send data to online monitor (first in the converter)
+                if self.socket:
+                    send_data(self.socket, data=np.array([measurement[0],measurement[1],measurement[2]], dtype=np.float64)) 
+
+                if write_flg == 1:
```

Comment:
Equivalent to `if write_flag`. And if this is a binary flag, you make it a boolean.